spikeinterface/core/baserecording.py

```diff
@@ -103,6 +104,13 @@ def get_traces(self,
             assert order in ["C", "F"]
             traces = np.asanyarray(traces, order=order)
         if return_scaled:
+            if hasattr(self, "NeoRawIOClass"):
+                if self.has_non_standard_units:
+                    message = ( 
```

Comment:
Why do you need parantesis here ?

Reply:
Make the message longer, not really needed as you don't have line length limits, right?